drivers/mfd/tswizard-core.c

```diff
@@ -7,28 +7,28 @@
 #include <linux/regmap.h>
 #include <linux/mfd/core.h>
 #include <linux/of_device.h>
-#include <linux/mfd/ts_supervisor.h>
+#include <linux/mfd/ts_wizard.h>
 
 #define MODEL_TS_7250_V3 0x7250
 
-static struct mfd_cell tssupervisor_devs[] = {
+static struct mfd_cell tswizard_devs[] = {
 	{
-		.name = "tssupervisor-reset",
+		.name = "tswizard-reset",
 		.id = -1,
 	},
 	{
-		.name = "tssupervisor-temp",
-		.of_compatible = "technologic,supervisor-temp",
+		.name = "tswizard-temp",
+		.of_compatible = "technologic,tswizard-temp",
 		.id = -1,
 	},
 	{
-		.name = "tssupervisor-adc",
-		.of_compatible = "technologic,supervisor-adc",
```

Comment:
I see this change goes from `technologic,supervisor-adc`

It looks like drivers/iio/adc/tswizard_adc.c has a conflicting compatible string, so that went from `technologic,tssupervisor-adc` to `technologic,tswizard-adc` but this is going from `technologic,supervisor-adc` to `technologic,tswizard-adc`

Based on this, it looks like we had very fragile naming, two drivers, one using `supervisor-adc` and another one using `tssupervisor-adc`, same with `-temp`. This should be looked at closer so we don't have conflicting or fragile compatible names.

Reply:
This *should* now be resolved by using tswizard-*  for all compatible strings. New commit posted.